op-conductor/rpc/ws/flashblocks_handler.go

```diff
@@ -148,33 +153,41 @@ func (h *Handler) BroadcastMessage(message []byte) {
 }
 
 func (h *Handler) startWebSocketServer(_ context.Context) error {
-	if h.cfg.WebsocketServerPort <= 0 {
-		return fmt.Errorf("WebSocket server port not configured or invalid: %d", h.cfg.WebsocketServerPort)
+	if h.cfg.WebsocketServerPort < 0 {
+		return fmt.Errorf("WebSocket server port invalid: %d", h.cfg.WebsocketServerPort)
 	}
-
 	h.hub = newHub(h.metrics)
 	go h.hub.run()
 
 	// Create HTTP server with WebSocket endpoint
 	mux := http.NewServeMux()
 	mux.HandleFunc("/ws", h.handleWebSocket)
 
-	// Start HTTP server
-	h.server = &http.Server{
-		Addr:    fmt.Sprintf(":%d", h.cfg.WebsocketServerPort),
-		Handler: mux,
+	// Start HTTP server using reusable httputil server (supports port=0 and exposes Port())
+	addr := fmt.Sprintf(":%d", h.cfg.WebsocketServerPort)
+	srv, err := httputil.StartHTTPServer(addr, mux)
+	if err != nil {
+		return err
+	}
+	h.httpServer = srv
+	// Determine bound port
+	if port, err := h.httpServer.Port(); err == nil {
+		h.boundPort = port
+	} else {
+		h.log.Error("failed to determine bound port", "err", err)
+		panic(err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to determine bound port: %w", err)
```
nit (apologies, now that I realise, it's better than panicking)